packages/amplify_datastore_plugin_interface/test/testData/Blog.dart

```diff
@@ -65,29 +76,32 @@ class Blog extends Model {
     var buffer = new StringBuffer();
 
     buffer.write("Blog {");
-    buffer.write("id=" + id + ", ");
-    buffer.write("name=" + name);
+    buffer.write("id=" + "$id" + ", ");
```

Comment:
*nit:* If you are going to use string interpolation
```dart
buffer.write("id=$id, ");
```

Reply:
good call. thanks!